arm-batch/2017-01-01/swagger/BatchManagement.json

```diff
@@ -1150,10 +1151,15 @@
         "id": {
```

Comment:
Please rename this to something else. property with name "id" inside of properties bag will throw a contract violation. Top level property names should not be repeated inside of properties bag. Something like "keyvaultId" would work.